docs/src/reference/index.md

```diff
@@ -1,5 +1,92 @@
+| [`tp.filter()`][temporian.filter]                       | Filters out events in a [`Node`][temporian.Node] for which a condition is false.            |
+| [`tp.glue()`][temporian.glue]                           | Concatenates [`Nodes`][temporian.Node] with the same sampling.                              |
+| [`tp.lag()`][temporian.lag]                             | Adds a delay to a [`Node`][temporian.Node]'s timestamps.                                    |
+| [`tp.leak()`][temporian.leak]                           | Subtracts a duration from a [`Node`][temporian.Node]'s timestamps.                          |
+| [`tp.prefix()`][temporian.prefix]                       | Adds a prefix to the names of the features in a [`Node`][temporian.Node].                   |
+| [`tp.propagate()`][temporian.propagate]                 | Propagates feature values over a sub index.                                                 |
+| [`tp.rename()`][temporian.rename]                       | Renames a [`Node`][temporian.Node]'s features and index.                                    |
+| [`tp.resample()`][temporian.resample]                   | Resamples a [`Node`][temporian.Node] at each timestamp of another [`Node`][temporian.Node]. |
+| [`tp.select()`][temporian.select]                       | Selects a subset of features from a [`Node`][temporian.Node].                               |
+| [`tp.since_last()`][temporian.since_last]               | Computes the amount of time since the last distinct timestamp.                              |
+| [`tp.tick()`][temporian.tick]                           | Generates timestamps at regular intervals in the range of a guide.                          |
+| [`tp.unique_timestamps()`][temporian.unique_timestamps] | Removes events with duplicated timestamps from a [`Node`][temporian.Node].                  |
+
+### Binary operators
+
+| Symbols                                                                                                                                                                                                                                           | Description                                                                   |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| [`tp.add()`][temporian.add] [`tp.subtract()`][temporian.subtract] [`tp.multiply()`][temporian.multiply] [`tp.divide()`][temporian.divide] [`tp.floordiv()`][temporian.floordiv] [`tp.modulo()`][temporian.modulo] [`tp.power()`][temporian.power] | Compute an arithmetic binary operation between two [`Nodes`][temporian.Node]. |
```

Comment:
nit: Add a line like:

Aliases for `+-*/`, etc.

I'm writing some examples there to cover cast, resample, etc, but I feel like it's hard to find if you're using arithmetic symbols instead.